### ISSUE TITLE
Activities heatmap

### ISSUE THREAD

**Issue by SamR1** [SamR1/FitTrackee#39]

**Comment by jat255** [comment]
Perhaps some existing code could be re-used to make this easier. 

- Here's a nifty JS example that's MIT-licensed: https://github.com/erik/derive
- Here's a Python one also MIT licensed: https://github.com/remisalmon/Strava-local-heatmap
- Another Python one with no license specified: https://github.com/TomCasavant/GPXtoHeatmap
- Another MIT Python: https://github.com/arichnad/heatmap-geojson

There are probably others out there, but this is what I found in a quick search.

**Comment by Hund** [comment]
This would be a really epic feature! I'm currently using statshunters.com for this, but that only supports Strava, and it's not self-hosted. :)

**Comment by TomCasavant** [comment]
> Perhaps some existing code could be re-used to make this easier.
> 
>     * Here's a nifty JS example that's MIT-licensed: https://github.com/erik/derive
> 
>     * Here's a Python one also MIT licensed: https://github.com/remisalmon/Strava-local-heatmap
> 
>     * Another Python one with no license specified: https://github.com/TomCasavant/GPXtoHeatmap
> 
>     * Another MIT Python: https://github.com/arichnad/heatmap-geojson
> 
> 
> There are probably others out there, but this is what I found in a quick search.


Yeah, I'd love this feature as well. I created that one without the license (and it now does have a license), but that one primarily uses javascript and google apis to create and generate the map and it works pretty slowly on larger sets, so I don't necessarily recommend mine as a basis for this
